src/fragment-components/a-tag.tsx

```diff
@@ -0,0 +1,192 @@
+import React from 'react';
+import {
+	Checkbox,
+	Dropdown,
+	Tag,
+	TextInput
+} from 'carbon-components-react';
+import { AComponent, ComponentInfo } from './a-component';
+import { ComponentCssClassSelector } from '../components/css-class-selector';
+
+import image from './../assets/component-icons/tag.svg';
+import {
+	angularClassNamesFromComponentObj,
+	nameStringToVariableString,
+	reactClassNamesFromComponentObj
+} from '../utils/fragment-tools';
+
+export const ATagStyleUI = ({selectedComponent, setComponent}: any) => {
+	const typeItems = [
+		{id: 'gray', text: 'Gray'},
+		{id: 'red', text: 'Red'},
+		{id: 'magenta', text: 'Magenta'},
+		{id: 'purple', text: 'Purple'},
+		{id: 'blue', text: 'Blue'},
+		{id: 'cyan', text: 'Cyan'},
+		{id: 'teal', text: 'Teal'},
+		{id: 'green', text: 'Green'},
+		{id: 'cool-gray', text: 'Cool gray'},
+		{id: 'warm-gray', text: 'Warm gray'},
+		{id: 'high-contrast', text: 'High contrast'},
+		{id: 'outline', text: 'Outline'},
+	];
+
+	const sizeItems = [
+		{id: 'md', text: 'Medium'},
+		{id: 'sm', text: 'Small'}
+	];
+
+	return <>
+		<TextInput
+			value={selectedComponent.title}
+			labelText='Title'
+			onChange={(event: any) => {
+				setComponent({
+					...selectedComponent,
+					title: event.currentTarget.value
+				});
+			}}
+		/>
+
+		<Dropdown
+			label='Type'
+			titleText='Type'
+			items={typeItems}
+			initialSelectedItem={typeItems.find(item => item.id === selectedComponent.kind)}
+			itemToString={(item: any) => (item ? item.text : '')}
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				kind: event.selectedItem.id
+			})}
+		/>
+
+		<Dropdown
+			label='Size'
+			titleText='Size'
+			items={sizeItems}
+			initialSelectedItem={typeItems.find(item => item.id === selectedComponent.size)}
+			itemToString={(item: any) => (item ? item.text : '')}
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				size: event.selectedItem.id
+			})}
+		/>
+
+		<Checkbox
+			labelText='Is filter'
+			id='filter'
+			checked={selectedComponent.filter}
+			onChange={(checked: boolean) => {
+				setComponent({
+					...selectedComponent,
+					filter: checked
+				});
+			}}
+		/>
+
+		<Checkbox
+			labelText='Disabled'
+			id='disabled'
+			checked={selectedComponent.disabled}
+			onChange={(checked: boolean) => {
+				setComponent({
+					...selectedComponent,
+					disabled: checked
+				});
+			}}
+		/>
+
+		<ComponentCssClassSelector componentObj={selectedComponent} setComponent={setComponent} />
+	</>
+};
+
+export const ATag = ({
+	children,
+	componentObj,
+	...rest
+}: any) => {
+	return (
+		<AComponent
+		componentObj={componentObj}
+		{...rest}>
+			<Tag
+			type={componentObj.kind}
```

Comment:
missing size?

(switching size in style context menu doesn't survive selecting another element and coming back - it goes to "Size" string)